src/lib/swapper/types.ts

```diff
@@ -236,50 +236,44 @@ export type CheckTradeStatusInput = {
 // no routes could be generated
 type TradeQuoteResult = Result<TradeQuote[], SwapErrorRight>
 
-type EvmTransactionRequestGas =
-  | {
-      gasPrice: string
-      maxFeePerGas?: never
-      maxPriorityFeePerGas?: never
-    }
-  | {
-      gasPrice?: never
-      maxFeePerGas: string
-      maxPriorityFeePerGas: string
-    }
-
 export type EvmTransactionRequest = {
-  nonce: string
+  nonce?: string
```

Comment:
Do we still want to pass this? The underlying implementation calls `adapter.buildCustomTx` which doesn't consume the custom nonce

Reply:
good call, i've removed it